shell/platform/darwin/ios/framework/Source/SemanticsObjectTest.mm

```diff
@@ -690,7 +690,7 @@ - (void)testShouldDispatchShowOnScreenActionForHidden {
   XCTAssertTrue(bridge->observations[0].action == flutter::SemanticsAction::kShowOnScreen);
 }
 
-- (void)testSemanticsObjectAndPlatformViewSemanticsContainerDontHaveRetainCycle {
```

Comment:
Not sure if this was ready to review, but this doesn't compile:
```
../../flutter/shell/platform/darwin/ios/framework/Source/SemanticsObjectTest.mm:699:54: error: no visible @interface for 'FlutterPlatformViewSemanticsContainer' declares the selector 'initWithSemanticsObject:'
      [[FlutterPlatformViewSemanticsContainer alloc] initWithSemanticsObject:object];
```

Reply:
sorry, it wasn't ready to review. I made it wip. Will mark as ready when it is ready.

Reply:
Sorry for getting ahead of myself then, thank you! 🙂